modules/core/test/test_mat.cpp

```diff
@@ -2356,4 +2356,95 @@ TEST(Mat, regression_18473)
 }
 
 
+TEST(Mat1D, basic)
```

Comment:
the test would be useful to move to separate PR, but mark it as disabled for now